docs/meshstack.customer-group-sync.md

```diff
@@ -169,6 +170,44 @@ let example =
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+#### IfNullAttributeTransformation
+
+<!--snippet:mesh.identityconnector.ldap.ifNullAttributeTransformation#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let IfNullAttributeTransformation =
+    {-
+      A IfNullAttributeTransformation takes a nullable attribute. If the value is null it uses a replacement value.
+      In a second step it applies an optional postProcessor on the value.
+
+      attribute:
+          The LDAP attribute that should be processed. Can be null. For example "mail"
```

Comment:
d: enabling null is a bit funny. I guess this would make this a nop in directly fallback to the ifNullAttribute?

Reply:
Thanks for the catch. That was formulated badly. The config value `null` does not have a special meaning, the attribute can be un-defined for an LDAP entity (e.g. an Service Account does not have a `mail` attribute). I updated the description.